test/testenv/verificationutils.go

```diff
@@ -676,3 +676,41 @@ func VerifyAppsDownloadedByInitContainer(deployment *Deployment, testenvInstance
 		}
 	}
 }
+
+// VerifyBundlePush verify that bundle push was pushed on all indexers
+func VerifyBundlePush(deployment *Deployment, testenvInstance *TestEnv, ns string, podName string, replicas int, previousBundleHash string) {
+	gomega.Eventually(func() bool {
+		// Get Bundle status and check that each pod has successfully deployed the latest bundle
+		clusterMasterBundleStatus := ClusterMasterBundlePushstatus(deployment, previousBundleHash)
+		if len(clusterMasterBundleStatus) < replicas {
+			testenvInstance.Log.Info("Bundle push on Pod not complete on all pods", "Pod with bundle push", clusterMasterBundleStatus)
+			return false
+		}
+		allPodNames := DumpGetPods(testenvInstance.GetName())
```

Comment:
A blind check for bundle on all indexers might cause some issues down the line. We should accept indexer pods as a parameter and remove this complexity from the method.